subworkflows/nf-core/gatk_paired_somatic_variant_calling/main.nf

```diff
@@ -0,0 +1,110 @@
+//
+// Run GATK mutect2 in tumour paired mode, getepileupsummaries, calculatecontamination, learnreadorientationmodel and filtermutectcalls
+//
+
+params.mutect2_options          = [:]
+params.learnorientation_options = [:]
+params.getpileup_options        = [suffix: '_tumour']
```

Comment:
I think in other processes we went along with `T` and `N`, didn't we @maxulysse ?

Reply:
or rather nothing and somatic ( in the case of somatic calling of course)